twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -289,12 +289,11 @@ def scrooge_scala_library(name,
       with_finagle = with_finagle,
   )
 
-  # deps from macro invocation would come via srcjar
-  # however, retained to make dependency analysis via aspects easier
   scala_library(
       name = name,
+      srcs = [srcjar],
```

Comment:
does this work? i think the scala_library rule is restricted to only allow actual files in the srcs not targets?

Reply:
(or does it intelligently realize the output file?... interesting)